docs/acs-private-apis.md

```diff
@@ -137,35 +137,24 @@ Get answer for a question.
 * Response:
     * Status: 200
     * Content-Type: `application/json`
-    * Entry object (wrapped with list/pagination information):
```

Comment:
There's only one answer for a question?

Reply:
In path `questions/{questionId}/answers` there shouldn't be also present `/-default-` ID?

Reply:
yep, from what I saw in code there is just one answer but there are multiple "Reference" inside

Reply:
fixed path